src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/SqlCommandIntegration.cs

```diff
@@ -343,7 +343,7 @@ public static class SqlCommandIntegration
                        .Start(moduleVersionPtr, mdToken, opCode, methodName)
                        .WithConcreteType(sqlCommandType)
                        .WithParameters(commandBehavior, cancellationToken)
-                       .WithNamespaceAndNameFilters(ClrNames.GenericTask, AdoNetConstants.TypeNames.CommandBehavior, ClrNames.CancellationToken)
+                       .WithNamespaceAndNameFilters($"{ClrNames.GenericTask}<{sqlDataReaderType.FullName}>", AdoNetConstants.TypeNames.CommandBehavior, ClrNames.CancellationToken)
```

Comment:
From looking at its source code, `Type.FullName` is not trivial. Instead of using `sqlDataReaderType.FullName`, consider reusing the result of `$"{sqlClientNamespace}.{SqlDataReaderTypeName}"` from line 327 above.

Reply:
Good idea, done in 70644ebb551fd0b8a466e760dd73af3df4a8ff10